lib/Unix/PID/Tiny.pm

```diff
@@ -242,9 +331,34 @@ Like Unix::PID but supplies only a few key functions.
 
 =head1 INTERFACE
 
-=head2 new()
+=head2 new(I<[$args_hr]>)
+
+See L<Unix::PID>'s new().  The following options can be provided in the
+optional HASH, I<$args_hr>, to enable certain extensions:
+
+=over
+
+=item B<keep_open>
+
+When a true value is provided, PID files created with pid_file() will remain
+open.
 
-See L<Unix::PID>'s new()
+=item B<check_proc_open_fds>
```

Comment:
FWiW, I think `=item` is already bolded do the `B<>` is superfluous

Reply:
It's not. I've attached a screenshot illustrating only keep_open being bolded.

![Screen Shot 2019-04-23 at 16 55 55](https://user-images.githubusercontent.com/234407/56618591-ac5b6680-65e8-11e9-9436-81a0f2ab729d.png)

Reply:
cool